incubator/openebs/templates/deployment-maya-provisioner.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "openebs.fullname" . }}-provisioner
+  namespace: {{ .Values.rbac.namespace }}
+  labels:
+    app: {{ template "openebs.name" . }}
+    chart: {{ template "openebs.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: provisioner
+spec:
+  replicas: {{ .Values.provisioner.replicas }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "openebs.name" . }}
+        release: {{ .Release.Name }}
+        component: provisioner
+    spec:
+{{- if .Values.rbac.create }}
```

Comment:
Remove if block and use `{{ template "openebs.serviceAccountName". }}`

Reply:
Removing the if block is failing my test. When I run the test with rbac.create=false, the template still fills in the serviceAccountName which is not created.

Reply:
My bad, the test was failing due to the bug - I was checking for rbac.create for creating serviceAccount. Fixed it.